<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndCruiseArea</class>
 <widget class="QWidget" name="WndCruiseArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WndCruiseArea</string>
  </property>
  <widget class="QLabel" name="id_lab_group_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="id_combox_cruise_num">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_cruise_start">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>40</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="WndPresetList" name="id_widget_image" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>69</y>
     <width>300</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WndPresetList</class>
   <extends>QWidget</extends>
   <header location="global">wndpresetlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
